bundles/org.openhab.ui/web/src/components/item/item-form.vue

```diff
@@ -174,8 +174,6 @@ export default {
   },
   mounted () {
     if (!this.item) return
-    if (!this.item.category) this.$set(this.item, 'category', '')
```

Comment:
@jimtng This fixes what you reported in another PR.
I have removed these two lines you added in your Item field alignment PR and haven't seen any misfunctionality (note that I updated toCode() to have them in the code tab, no matter they are defined or not.)

Reply:
IIRC the item.category was already there, I added the second line. Removing them here is better anyway, because this file isn't responsible for this.item. The caller of this component should've fixed that up before passing `item` to it.

Reply:
Missing category and groupName don't seem to be a problem to me, I see no warnings logged or UI misfunctions.

Reply:
I'm guessing that it might have only been a problem in the data() which you fixed by adding `|| []`

Reply:
Yes, seems so.
I will merge this and we will see if any new problems arise.